pkg/pillar/cmd/cpuallocator/cpuallocator.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2022 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Do we want to make some kind of command-line utility later? If not, please move this package out of cmd directory. We have different logic in [Makefile](https://github.com/lf-edge/eve/blob/1844bc4b5a825fd33a06b7630136a6cbc6e3797d/pkg/pillar/Makefile#L21) for subdirectories of cmd and make symlinks to zedbox.

Reply:
Can you explain the logic? What is the purpose of the cmd directory? Where should it go then?

Reply:
Sure, cmd directory contains packages which has [Run](https://github.com/lf-edge/eve/blob/1844bc4b5a825fd33a06b7630136a6cbc6e3797d/pkg/pillar/cmd/baseosmgr/baseosmgr.go#L79) function to be pointed from [zedbox](https://github.com/lf-edge/eve/blob/1844bc4b5a825fd33a06b7630136a6cbc6e3797d/pkg/pillar/zedbox/zedbox.go#L97). When we build pillar we actually build only zedbox binary and make symlinks to zedbox with names from every package inside cmd directory. When we run i.e. baseousmgr (actually it is symlink), it is resolved to zedbox, but have baseosmgr in command name. We [parse](https://github.com/lf-edge/eve/blob/1844bc4b5a825fd33a06b7630136a6cbc6e3797d/pkg/pillar/zedbox/zedbox.go#L113) command name from zedbox binary and start what we expect to start using that information.
So cmd directory is the place we want to see special prepared packages which are in entrypoint list of zedbox.
In your case you can put the code into pkg/pillar/cpuallocator.

Reply:
done